packages/components/src/components/button/button.jsx

```diff
@@ -59,8 +66,9 @@ const Button = ({
 Button.propTypes = {
     children         : PropTypes.node,
     className        : PropTypes.string,
-    classNameSpan    : PropTypes.string,
+    classNameSpa     : PropTypes.string,
     has_effect       : PropTypes.bool,
+    Icon             : PropTypes.node,
     id               : PropTypes.string,
     is_disabled      : PropTypes.oneOfType([PropTypes.number, PropTypes.bool]),
```

Comment:
Shouldn't this always be a boolean?

Reply:
@aaron-binary i havent add it, no idea. i think someone is using it this way. lets keep it